swan-lake/learn/mocking.md

```diff
@@ -34,32 +34,41 @@ You can write a custom mock object and substitute it in place of the real object
 Let's make changes to the example in the [Quick Start](/swan-lake/learn/testing-quick-start) page. In order to test the
  `getRandomJoke` function without actually calling the endpoint, use the `test:mock` function to mock the `get` remote function of the client object.
 
-Change the content of the ***main_test.bal*** file as follows:
+Change the content of the test file as follows:
+
+This is the definition of the mock object which can be used as the test double for the `clientEndpont` object. Only the `get` function is implemented since it is the only function used in the sample.
```

Comment:
```suggestion
This is the definition of the mock object, which can be used as the test double for the `clientEndpont` object. Only the `get` function is implemented since it is the only function used in the sample.
```